### ISSUE TITLE
Is there a way to access gradients of each layer?

### ISSUE THREAD

**Issue by marcel-proust** [fchollet/keras#115]
I want to display the distribution of gradients of each layer. How can I do it?

**Comment by fchollet** [comment]
Gradients are handled by Theano, and there is no practical way to access them.

**Comment by stepelu** [comment]
Would that be true even when using Keras on top of Tensorflow?

**Comment by raginisharma14** [comment]
Even I am looking for the same thing. Is it true even if we use keras on top of tensorflow
